cmd/plugin/plugin.go

```diff
@@ -29,39 +29,88 @@ func NewCommand(out output.Output) *cobra.Command {
 
 // InstallCommand creates a new command to install
 func InstallCommand(out output.Output) *cobra.Command {
-	return &cobra.Command{
+	var (
+		pluginSource  string
```

Comment:
We should remove it since it does not any function

Reply:
👍